rally_ovs/plugins/ovs/scenarios/ovn_network.py

```diff
@@ -88,7 +88,16 @@ def create_and_bind_ports(self,
 
         sandboxes = self.context["sandboxes"]
 
-        lswitches = self._create_networks(network_create_args)
+        # For scale test, we need to also add more ports on demand by adding
+        # new HVs. Hence, we can skip creating lswitches and lrouters since
+        # we just need lports.
+        use_existing_networks = port_create_args. \
+            get("use_existing_networks", False)
+        if not use_existing_networks:
+            lswitches = self._create_networks(network_create_args)
+        else:
+            lswitches = self._list_lswitches()
```

Comment:
_list_lswitches only return ls names, but here we need not only names but also CIDRs. Without CIDRs, lports won't have IP addresses and the ARP flows won't be created, and so the test result can be different from production environment because the number of flows will be much less. We need to solve this problem.

Reply:
Thats take care!